tiledb/api/c_api/dimension_label/dimension_label_api_external.h

```diff
@@ -53,9 +53,7 @@ typedef struct tiledb_dimension_label_handle_t tiledb_dimension_label_t;
  *
  * @code{.c}
  * tiledb_dimension_label_t* dim_label;
- * tiledb_array_schema_get_dimension_label_from_name(
- *       ctx, array_schema, "label1", &dimension_label);
- * tiledb_dimension_label_free(&dimension_label);
+ * tiledb_dimension_label_free(&dim_label);
```

Comment:
This change removes a useful example. The documentation was already confusing by being too terse. Better to expand it here, say by making it clear that `tiledb_array_schema_get_dimension_label_from_name` is one of the C API functions that yields a handle that must be freed, and that it's exemplary here and not the only one that does so.